### ISSUE TITLE
Instructor file editing crashes server (v2)

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#1708]
Current `master` (with #1707 merged) on `pl-dev` crashes when attempting to edit a file:

```
/PrairieLearn/node_modules/aws-sdk/lib/request.js:31
throw err;
^
Error [TypeError]: string.charCodeAt is not a function
at ucs2decode (/PrairieLearn/node_modules/utf8/utf8.js:14:19)
at Object.utf8encode [as encode] (/PrairieLearn/node_modules/utf8/utf8.js:88:20)
at Object.module.exports.b64EncodeUnicode (/PrairieLearn/lib/base64-util.js:5:31)
at Response.<anonymous> (/PrairieLearn/pages/instructorFileEditor/instructorFileEditor.js:309:57)
at Request.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/request.js:364:18)
at Request.callListeners (/PrairieLearn/node_modules/aws-sdk/lib/sequential_executor.js:106:20)
at Request.emit (/PrairieLearn/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
at Request.emit (/PrairieLearn/node_modules/aws-sdk/lib/request.js:683:14)
at Request.transition (/PrairieLearn/node_modules/aws-sdk/lib/request.js:22:10)
at AcceptorStateMachine.runTo (/PrairieLearn/node_modules/aws-sdk/lib/state_machine.js:14:12)
at /PrairieLearn/node_modules/aws-sdk/lib/state_machine.js:26:10
at Request.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/request.js:38:9)
at Request.<anonymous> (/PrairieLearn/node_modules/aws-sdk/lib/request.js:685:12)
at Request.callListeners (/PrairieLearn/node_modules/aws-sdk/lib/sequential_executor.js:116:18)
at Request.emit (/PrairieLearn/node_modules/aws-sdk/lib/sequential_executor.js:78:10)
at Request.emit (/PrairieLearn/node_modules/aws-sdk/lib/request.js:683:14) {
message: 'string.charCodeAt is not a function',
code: 'TypeError',
time: 2019-09-16T12:59:45.424Z
}
```

**Comment by mwest1066** [comment]
I suspect that this issue is showing up here because CI tests use local file storage, while prod uses S3. We should change to use `lib/file-store.js` in both cases, so that CI and prod are identical.

This will probably require changing `file-store.upload()` to return the `file_id` of the newly-added file (the sproc already returns `file_id`, we just don’t pass it back in the JS function).

**Comment by mwest1066** [comment]
Fixed by #1711
